module/documents/item.mjs

```diff
@@ -1451,6 +1451,33 @@ export default class Item5e extends SystemDocumentMixin(Item) {
 
   /* -------------------------------------------- */
 
+  /** @inheritDoc */
+  _onUpdate(changed, options, userId) {
```

Comment:
This is restoring functionality that was lost from the old enchantment config. We need to flag AEs as riders, otherwise they affect the Actor when they're supposed to just be stored on the Item for use as part of an enchantment.